Formula/skktools.rb

```diff
@@ -24,4 +24,65 @@ def install
     ENV.deparallelize
     system "make", "install"
   end
+
+  test do
+    test_dic = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるk /悪/
+      わるi /悪/
+    EOS
+    (testpath/"SKK-JISYO.TEST").write test_dic
+
+    test_shuffle = <<~EOS.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるi /悪/
+      わるk /悪/
+    EOS
+
+    expect_shuffle = <<~EOS.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+      わるi /悪/
+    EOS
+
+    test_sp1 = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるs /悪/
+      わるk /悪/
+    EOS
+    (testpath/"test.sp1").write test_sp1
+
+    test_sp2 = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるk /悪/
+      わるi /悪/
+    EOS
+    (testpath/"test.sp2").write test_sp2
+
+    test_sp3 = <<~EOS.strip.tap { |s| s.encode("euc-jis-2004") }
+      わるi /悪/
+    EOS
+    (testpath/"test.sp3").write test_sp3
+
+    expect_expr = <<~EOS.tap { |s| s.encode("euc-jis-2004") }
+      ;; okuri-ari entries.
+      わるs /悪/
+      わるk /悪/
+    EOS
+
+    # test skkdic-count
+    expect_count = "SKK-JISYO.TEST: 3 candidates\n"
+    actual_count = pipe_output("#{bin}/skkdic-count SKK-JISYO.TEST", nil, 0)
```

Comment:
You can still use `shell_output()` on this one. Just meant where we were piping things it made sense to use `pipe_output()`. Apologies if my prior wording was confusing!

Reply:
> You can still use shell_output() on this one, I think? Just meant where we were piping things it made sense to use pipe_output().

I fixed it, thank you!